<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SpatialDecisionDockWidgetBase</class>
 <widget class="QDockWidget" name="SpatialDecisionDockWidgetBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>371</width>
    <height>509</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SDSS Template</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTabWidget" name="sampleWidgets">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="dataTab">
       <attribute name="title">
        <string>Input Data</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,0,0">
        <item>
         <layout class="QFormLayout" name="formLayout">
          <item row="0" column="0">
           <widget class="QPushButton" name="openScenarioButton">
            <property name="text">
             <string>Open Scenario</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QPushButton" name="saveScenarioButton">
            <property name="text">
             <string>Save Scenario</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QGridLayout" name="gridLayout_3" columnstretch="0,0">
          <item row="0" column="1">
           <widget class="QComboBox" name="selectLayerCombo"/>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="selectLayerLabel">
            <property name="text">
             <string>Select transit layer:</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="AnalysisTab">
       <attribute name="title">
        <string>Analysis</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLabel" name="UserGroupLabel">
          <property name="text">
           <string>Select user group</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="UserGroupCombo">
          <item>
           <property name="text">
            <string>Students</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Adults</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Elderly</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="BuffersLabel">
          <property name="text">
           <string>1. Create buffers of all transit nodes </string>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="BuffersButton">
          <property name="text">
           <string>Create Buffers</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="FindNodesLabel">
          <property name="text">
           <string>2. Create heatmap</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="FindNodesButton">
          <property name="text">
           <string>Create heatmap</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="NoServiceLabel">
          <property name="text">
           <string>3. Find areas without service</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="NoServiceButton">
          <property name="text">
           <string>Find no-service areas</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="AddNodeLabel">
          <property name="text">
           <string>4. Add new node</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="AddNodeButton">
          <property name="text">
           <string>Add node</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Placeholder</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Visualisation</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="ReportingTab">
       <attribute name="title">
        <string>Reporting</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,0">
        <item>
         <layout class="QGridLayout" name="gridLayout_2" rowstretch="0,1,1" rowminimumheight="0,1,1">
          <item row="0" column="0">
           <widget class="QLabel" name="featureCounterLabel">
            <property name="text">
             <string>Feature counter:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QPushButton" name="featureCounterUpdateButton">
            <property name="text">
             <string>Update</string>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QPushButton" name="saveMapPathButton">
            <property name="text">
             <string>...</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QPushButton" name="saveMapButton">
            <property name="text">
             <string>Save Map</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0" colspan="2">
           <widget class="QLineEdit" name="saveMapPathEdit"/>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="featureCounterEdit"/>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QListWidget" name="reportList"/>
        </item>
        <item>
         <widget class="QTableWidget" name="statisticsTable"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
